tests/flow/transform/test_google_flow.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+from unittest.mock import MagicMock, patch
+from uniflow.flow.transform.transform_google_flow import GoogleModelFlow, TransformGoogleFlow
+from uniflow.node import Node
+from uniflow.op.prompt import PromptTemplate
+from uniflow.op.model.lm.model import LmModel
+
+class TestGoogleModelFlow(unittest.TestCase):
+    @patch('uniflow.flow.transform.transform_google_flow.LmModel')
+    @patch('uniflow.flow.transform.transform_google_flow.ModelOp')
+    def setUp(self, mock_model_op, mock_lm_model):
+        self.mock_model_op = mock_model_op
+        self.mock_lm_model = mock_lm_model
+        self.mock_lm_model.return_value = MagicMock()
+        self.prompt_template = PromptTemplate(instruction="instruction", few_shot_prompt=[{}])
+        self.model_config = {"response_format": {"type": "json_object"}}
+        self.google_flow = GoogleModelFlow(self.prompt_template, self.model_config)
```

Comment:
same above, we should add two more unit tests to test with prompt_template and mode_config are none

Reply:
we can control using python type hints to enforce the none check in the constructor.  Let me figure out how to set it up